generative_ai/gemini_count_token_example.py

```diff
@@ -58,7 +58,8 @@ def count_tokens_multimodal(project_id: str) -> GenerationResponse:
 
     vertexai.init(project=project_id, location="us-central1")
 
-    model = GenerativeModel(model_name="gemini-1.5-pro-preview-0409")
+    # For Gemini supported models check https://cloud.google.com/vertex-ai/generative-ai/docs/model-reference/inference
```

Comment:
Let's not put this comment. The page where the sample is shown should have the model information.